chat_caller.py

```diff
@@ -14,8 +14,7 @@
 
 load_dotenv()
 isDocker = os.path.exists("/.dockerenv")
-if isDocker:
```

Comment:
Received the following error when attempting to run the docker container with the absolute path 
<img width="866" alt="Screenshot 2024-06-10 at 11 21 17 AM" src="https://github.com/MuhammadIrfanAli/pruju-ai/assets/169608507/5e00c7fe-fe18-4a09-b23d-fa1c708e3a6b">